backend/app/services/helpOffers/remove.ts

```diff
@@ -0,0 +1,11 @@
+import { AppError } from "../../common/errors/AppError"
+import * as helpOffersRepo from "../../repositories/helpOffers/remove"
+import { errors } from "../../common/errors/errors"
+
+export const remove = async(help_request_id:number, AuthUserId: number) => {
+    const helpOffer = await helpOffersRepo.remove(help_request_id, AuthUserId)
+    if(helpOffer === undefined){
```

Comment:
it's not clear here what;s inside, usually query return has rowCount props which indicated where the elem;s been deleted. I don't think ur check works here bc it is always defined